packages/providers/src/lib/mobishastra/mobishastra.provider.ts

```diff
@@ -0,0 +1,57 @@
+import {
+  ChannelTypeEnum,
+  ISendMessageSuccessResponse,
+  ISmsOptions,
+  ISmsProvider,
+} from '@novu/stateless';
+import { v4 as uuid } from 'uuid';
+import axios, { AxiosInstance } from 'axios';
+
+export class MobishastraProvider implements ISmsProvider {
+  id = 'mobishastra';
+  channelType = ChannelTypeEnum.SMS as ChannelTypeEnum.SMS;
+  axiosInstance: AxiosInstance;
+  headers: Record<string, string>;
+
+  constructor(
+    private config: {
+      baseUrl: string;
+      username: string;
+      password: string;
+      language?: string;
+      from: string;
+    }
+  ) {
+    this.axiosInstance = axios.create({
+      baseURL: config.baseUrl,
+      headers: {
+        'Content-Type': 'application/json',
+      },
+    });
+  }
+
+  async sendMessage(
+    options: ISmsOptions
+  ): Promise<ISendMessageSuccessResponse> {
+    const response = await this.axiosInstance.request({
```

Comment:
Please add the necessary error handling.

Reply:
DOne